src/Components/Media_Container/ListView.js

```diff
@@ -0,0 +1,207 @@
+import React, { useState, useRef, useEffect } from "react";
+import playBtn from "../../Images/playBtn.png";
+import styled from "styled-components";
+import play from "../../Images/play.png";
+import pause from "../../Images/pause.png";
+import next from "../../Images/next.png";
+import prev from "../../Images/prev.png";
+
+const StyledWrapper = styled.div`
+  margin: 0 auto;
+  padding: 15px 20px;
+  box-sizing: border-box;
+  margin-top: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  cursor: pointer;
+`;
+const WrapperItems = styled.div`
+  border: 1px solid;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  border-radius: 15px;
+  overflow: hidden;
+  width: 200px;
+`;
+const ImgBlock = styled.div`
+  background: red;
+  margin: 0 auto;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+`;
+const ContentBlock = styled.div`
+  padding: 15px 0px;
+`;
+const PlayBtn = styled.div`
+  position: absolute;
+  width: 35%;
+`;
+const PlaySection = styled.div`
+  /* audio {
+    outline: none;
+    width: 95%;
+    bottom: 5%;
+    position: absolute;
+    margin: 0 auto;
+    left: 50%;
+    transform: translateX(-50%);
+  } */
+
+  width: 95%;
+  height: 54px;
+  bottom: 5%;
+  position: absolute;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  -ms-transform: translateX(-50%);
+  transform: translateX(-50%);
+  border-radius: 50px;
+  background: #e1e3e4;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  box-sizing: border-box;
+  padding: 0px 25px;
+`;
+const ControlSection = styled.div`
+  img {
+    width: 15% !important;
+    cursor: pointer;
+  }
+  /* border: 1px solid; */
+  width: 10%;
+  height: 70%;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  overflow: hidden;
+`;
+const TimeSection = styled.div`
+  /* border: 1px solid; */
+  width: 5%;
+  height: 70%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Progressbar = styled.div`
+  width: 65%;
+  height: 7%;
+  border-radius: 50px;
+  background: #908e8e;
+  padding: 1px 0px;
+  margin-left: 1%;
+`;
+
+const Progress = styled.div`
+  width: 15px;
+  height: 15px;
+  background: #000;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+`;
+
+const ListView = ({ data }) => {
+  const [activeTab, setActiveTab] = useState(null);
+  const [playing, setPlaying] = useState(false);
+  const [ButtonPlay, setButtonPlay] = useState(play);
```

Comment:
wrong variable name should be `playerState` it's value should be either `PLAYING`, `PAUSED`

create a object 
```
const PlayerState = {
  PLAYING = 'PLAYING',
PAUSED = 'PAUSED'
}
```

and create state like this  `const [playerState,setPlayerState ] = useState(PlayerState.PAUSED)`